ingest-app/src/cmr/ingest/api/bulk.clj

```diff
@@ -38,13 +38,17 @@
   (generate-xml-status-list result status-list-key status-key id-key nil))
  ([result status-list-key status-key id-key created-at-key]
   (generate-xml-status-list result status-list-key status-key id-key created-at-key nil))
- ([result status-list-key status-key id-key created-at-key additional-keys]
+ ([result status-list-key status-key id-key created-at-key name-key]
+  (generate-xml-status-list result status-list-key status-key id-key created-at-key name-key nil))
+ ([result status-list-key status-key id-key created-at-key name-key additional-keys]
```

Comment:
Nitpick: I feel like the arity of this function is a little out of control. I'm not sure how we would fix it, but I think it's worth establishing how to break something like this up. I'm thinking @oubiwann might have some thoughts here

Reply:
I looked at the full namespace. It is a private function that is used in exactly two places. So there definitely should not be more than 2 arrities.

There is not that much code that would be repeated, so it makes sense to just break into two simple functions that just take the result as an argument.